### DISCUSSION TITLE
Blue notification dot 'stuck'

### DISCUSSION THREAD

**Discussion by fyrak1s** [community/community#123129] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello, since a few months my notification dot is stuck, even if I have read all the notifications

Would any dev be able to look into it? I am not the only one reporting this issue.

Thanks:)

**Comment by iTechTop** [comment]
👋 If the red notification icon stays visible despite reading all notifications, it may indicate a bug or a discrepancy in handling notification states. Though I can't physically examine your UI, I can provide some tips to help resolve the issue:

1. Force refresh the webpage by pressing `Ctrl+F5` or `Shift+F5` on your keyboard to eliminate client-side caching issues.
2. Clear your browser cache and delete cookies for the website. Be aware that clearing the cache will sign you out, forcing you to re-login.
3. Log out and exit the browser, then wait a few minutes before logging back in. Sometimes, waiting briefly can help synchronize notification data.
4. Browse the site in a private/incognito window. This eliminates interference from browser extensions and saves your browsing data.
5. If you're using bookmarklets or plugins that affect GitHub, try disabling them momentarily to rule out conflicts.
6. Reset your notification settings by turning off all notification types, saving the changes, and then re-enabling them.

When numerous users encounter this problem, it suggests an underlying issue affecting multiple users simultaneously. Notify GitHub Support about the problem by opening an issue ticket on GitHub Enterprise Cloud Issue Tracker: https://help.github.com/enterprise/admin/guides/installation-configuration-and-troubleshooting/creating-an-issue-for-github-entertainment-cloud/.

Be prepared to supply requested diagnostic information to aid the investigation, such as your browser type and version, OS, affected pages, and reproduction steps. Following these steps should help you resolve the issue. Patience and cooperation are vital during debugging, as fixing bugs takes time and thorough analysis.

**Comment by ebndev** [comment, marked as answer]
Hey @Nuc1eoN! Here are 2 solutions from previous Discussions. Hopefully one of them works for you!

Simple: https://github.com/orgs/community/discussions/32380#discussioncomment-3785542
Curl command: https://github.com/orgs/community/discussions/6874#discussioncomment-2859125

**Reply by fyrak1s** [reply]
The curl command from https://github.com/orgs/community/discussions/6874#discussioncomment-9387397 fixed it thanks!

**Reply by fyrak1s** [reply]
Here is a concrete step by step how to do it if anybody needs it: 

> You have to [create a GitHub token](https://github.com/settings/tokens/new), select "Access notifications" and create it.
> 
> Copy the token and in your console run `export TOKEN=<your-token>`.
> 
> Then copy and paste this into your console and run it:
> ```bash
> curl -X PUT \
>    -H "Accept: application/vnd.github.v3+json" \
>    -H "Authorization: token $TOKEN" \
>     https://api.github.com/notifications \
>    -d @<(cat <<EOF
>    {
>      "last_read_at": "$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
>    }
> EOF
> )
> ```
https://github.com/orgs/community/discussions/43032#discussioncomment-9456373

**Reply by ebndev** [reply]
Nice! Thanks for sharing the link to that comment 😄

**Reply by PavanLuca** [reply]
Why users have to create tokens or code if the problem is well-known? Doesn't it have to be fixed on GitHub code? Or even add this token as default for everyone?
![immagine](https://github.com/community/community/assets/123581658/2f80c527-a7ea-4d1e-9fb5-6274b6f1c8a0)

**Reply by sepehrkiller** [reply]
because Github lacks competence as any other big company do, people working in big companies are not dedicated to making the best App, they are just doing whatever they are told to do which lacks soul and dedication

you would be the same if you worked in these environments

websites, apps, software, ... everything has gotten and is getting worse and worse over time

I refuse to fix the notification bug by running a curl command, it's not my job to fix this bug, it's Github's
maybe we should make some accounts and spam these kinds of things daily so that they will fix it? seems like the only option (just kidding)

if Github is not going to fix bugs, then why can we even report them? this is literally the simplest thing to fix
